src/components/PropList.js

```diff
@@ -179,4 +256,211 @@ PropList.propTypes = {
   ),
 };
 
+export const query = graphql`
+  fragment PropList_propTypes on NewRelicSdkPropTypeDefinition {
```

Comment:
Don't ask. This query was so hard 😂

Reply:
I remember trying to write prop types for the prop prop types and getting so turned around 😵